README.example.md

```diff
@@ -77,69 +66,30 @@ stub any external calls using the test suite's features.
 Setting up the configuration is a matter of copying the supplied examples and
 filling in the gaps, as follows:
 
+1. `cp .env.example .env`
+2. `cp .env.development.example .env.development`
+3. Fill in env vars that are local to your setup
 
-## Requirements
-
-You must have the following installed and available on your machine:
-
-- **Ruby 2.6.5**
-- **Node JS 12.x**
-- **PostgreSQL 11.3**
-
-## Set up the app the first time
-```sh
-$ cp .env.example .env
-# edit .env in your editor of choice
-
-$ cp .env.development.example .env.development
-# edit .env.development in your editor of choice
-```
-
-Install the Ruby and JS dependencies
-
-```sh
-$ bundle install
-$ npm install -g yarn
-$ yarn install
-```
-
-Prepare the database
-
-You can set `DATABASE_URL` in `.env.local`, if for instance you use Docker for Postgres. `DATABASE_URL="postgresql://localhost:5432"`
-```sh
-$ bundle exec rails db:setup
-```
-
-Download the correct version of chromedriver, it will try to do it automatically, but if you disable network with webmock/vcr your tests will fail when it does.
-```sh
-$ bundle exec rails webdrivers:chromedriver:update
-```
-
-Run the test suite and expect all tests to pass
-
-```sh
-$ bundle exec rspec
-```
-
-Start a development server and check out the app in a browser
+### Dependencies and database setup
 
-```sh
-heroku local
-```
+Run: `bin/setup`
 
-## Run the app
+## Day-to-day
 
-`bundle exec rails s`
+- Run the server: `heroku local` and [http://localhost:5000](http://localhost:5000)
```

Comment:
are we actually supposed to run it with `heroku local`? 🤔

Reply:
I have never used it.
How many of you are using `heroku local`, `foreman` or the like?
How well does it combine with using `byebug`?

Reply:
I decided to go with `heroku local` because it was mentioned before `bundle exec rails s`, and because I think that the person who decided to add `heroku local` must have had an opinion about it.

Personally I don't care a lot.